### DISCUSSION TITLE
Computational Plasma Project

### DISCUSSION THREAD

**Discussion by ycaophysics** [williamgilpin/cphy#25] [projects]
Anyone interested in doing computational plasma physics as the project? (Topics include general plasma physics turbulences, laser plasma interaction etc)

Yuxuan Cao

**Comment by abao1999** [comment]
This sounds really cool! Any datasets you're thinking about?

**Comment by abao1999** [comment]
@Art-of-FUgue (Hongming)

**Comment by abao1999** [comment]
http://case.physics.stonybrook.edu/images/7/76/Transverse_beam_dynamics_and_emittance_preservation.pdf
